sympy/simplify/simplify.py

```diff
@@ -264,7 +264,7 @@ def posify(eq):
     [2]
     """
     eq = sympify(eq)
-    if iterable(eq):
+    if iterable(eq) and not isinstance(eq, Basic):
```

Comment:
I would reverse the order of these two checks because the `isinstance` case will almost always be False and is faster.